app/views/layouts/application.html.erb

```diff
@@ -62,31 +62,37 @@
         <div class="col-3"></div>
         <div class="col-6">
           <br>
+          <!-- text-align instead of align because align is depracted and no longer works-->
```

Comment:
No need to add your comments in the code, simply describe it in a comment on your change in the PR (like I am right now) or explain it in the PR description.
```suggestion
```